test/manual/TokenMetadataFetchFailed.md

```diff
@@ -0,0 +1,65 @@
+
+# Error reported in API when failing to fetch metadata
+
+## OS
+
+Any
+
+## Steps
+
+1. Start cardano-node and cardano-wallet on `testnet` using latest [config](https://hydra.iohk.io/job/Cardano/iohk-nix/cardano-deployment/latest/download/1/index.html). Make sure both are fully synced.
+
+```bash
+$ cardano-node run \
+		--config ./*-config.json \
+		--topology ./*-topology.json \
+		--database-path ./db \
+		--socket-path ./node.socket
+
+$ cardano-wallet serve --port 8090 \
+		--node-socket ../relay1/node.socket \
+		--testnet testnet-byron-genesis.json  \
+		--database ./wallet-db \
+		--token-metadata-server http://localhost/
+```
+Make sure wallet server `--token-metadata-server` points to an url which is not token metadata server.
+
+2. On the wallet that has some assets check that `"metadata_error":"fetch"` is returned when getting assets.
+ - when listing all assets
+```
+$ curl -X GET http://localhost:8090/v2/wallets/73d38c71e4b8b5d71769622ab4f5bfdedbb7c39d/assets
```

Comment:
Could we mention the public key for this wallet such that anyone can run this?

Reply:
That's a nice idea.

In the meantime I have added instructions for how to get one of our E2E fixture wallets, using the password from our shared LastPass folder.